### ISSUE TITLE
You shouldn't be able to do anything with `Any`

### ISSUE THREAD

**Issue by not-my-profile** [microsoft/pyright#3148]
```python
from typing import Any

Any + Any
Any[123]
```

Pyright doesn't complain about either expression although both are runtime errors.

**Comment by erictraut** [comment]
I'm not sure if it's worth flagging this as an error. This usage is so obviously wrong that I don't think anyone would ever attempt to use `Any` in this manner.

**Comment by erictraut** [comment]
I'm going to close for now. If it comes up again, I might consider it.
